src/components/ChartSetting.jsx

```diff
@@ -105,6 +107,15 @@ const ChartSetting = ({
                             />
                         </div>
                     </div>
+                    <div className="ciq-list-item">
+                        <span className="ciq-icon-text">{t.translate('Auto Scale')}</span>
+                        <div className="ciq-action">
+                            <Switch
+                                value={isAutoScale}
+                                onChange={setAutoScale}
+                            />
+                        </div>
+                    </div>
```

Comment:
@morteza-binary i think its better to sort settings alphabetically since its getting big. cause now they has no meaningful order.